Sources/Core/ObjectiveCIR.swift

```diff
@@ -379,11 +432,21 @@ public struct ObjCIR {
                     }).joined(separator: "\n"),
                     "@end",
                 ].map { $0.trimmingCharacters(in: CharacterSet.whitespaces) }.filter { $0 != "" }
-            case let .category(className: className, categoryName: categoryName, methods: methods, properties: properties):
+            case let .category(className: className, categoryName: categoryName, methods: methods, properties: properties, variables: variables):
                 // Only render anonymous categories in the implementation
                 guard categoryName == nil else { return [] }
+                let variableDeclarations: String
+                if !variables.isEmpty {
+                    let vars: [String] = variables.map { (param, typeName) -> String in
+                        "\t\(typeName) _\(Languages.objectiveC.snakeCaseToPropertyName(param));"
+                    }
+                    variableDeclarations = (["{"] + vars + ["}"]).joined(separator: "\n")
+                } else {
+                    variableDeclarations = ""
```

Comment:
Will this result in more empty lines?

Reply:
No. Please see the examples `Board`, `Image`, `Model`, `VariableSubstitution`. They do not have the variable section. However `Everything`, `Pin`, and `User` do.